src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cn from 'classnames';
+import { FilterBy } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filterBy: FilterBy,
+  todos: Todo[],
+  setFilterBy: React.Dispatch<React.SetStateAction<FilterBy>>,
+};
+
+export const Footer: React.FC<Props> = ((props) => {
+  const { filterBy, todos, setFilterBy } = props;
+
+  const countUncompletedTodo = todos.filter(todo => !todo.completed).length;
```

Comment:
you can also use useMemo() hook to prevent overcomputing, something like:
```
const countUncompletedTodo = useMemo(() => {
  return todos.filter(todo => !todo.completed).length;
}, [todos]);
```

Reply:
thank you for your insight, I'm gonna apply it if necessary on later stages of this project